Error when building.

## Description

Fails on build with Maven, may it be related to https://github.com/cobolbaby/kafka-connect-rabbitmq/commit/1cbce8dfd314bd636ec4e5ae32c27c5f3e4a210d

## Reason

Can't find dependencies:

```
[ERROR] Failed to execute goal on project kafka-connect-rabbitmq: Could not resolve dependencies for project com.themeetgroup.kafka.connect:kafka-connect-rabbitmq:jar:0.1.0: Failed to collect dependencies at io.confluent:kafka-avro-serializer:jar:5.1.0: Failed to read artifact descriptor for io.confluent:kafka-avro-serializer:jar:5.1.0: Could not transfer artifact io.confluent:kafka-avro-serializer:pom:5.1.0 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [confluent (http://packages.confluent.io/maven/, default, releases+snapshots)] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

## Steps to reproduce

```
git clone https://github.com/cobolbaby/kafka-connect-rabbitmq.git /kafka-connect-rabbitmq
cd /kafka-connect-rabbitmq && mvn install
```

--- a/pom.xml
+++ b/pom.xml
@@ -76,7 +76,27 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-avro-serializer</artifactId>
-            <version>5.1.0</version>
+            <version>5.1.4</version>
+        </dependency>
+        <dependency>
+            <groupId>com.palantir.docker.compose</groupId>
+            <artifactId>docker-compose-junit-jupiter</artifactId>
+            <version>1.7.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.palantir.docker.compose</groupId>
+            <artifactId>docker-compose-rule-junit4</artifactId>
+            <version>1.7.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.palantir.docker.compose</groupId>
+            <artifactId>docker-compose-rule-core</artifactId>
+            <version>1.7.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.jcustenborder</groupId>
+            <artifactId>docker-compose-junit-extension</artifactId>
+            <version>0.1.6</version>
         </dependency>
     </dependencies>
     <build>

